test/authextension.test.ts

```diff
@@ -38,6 +38,10 @@ suite('authextension Modules', () => {
     }
   };
 
+  enum GlobalState {
+    UUID = 'uuid'
+  }
+
```

Comment:
Why can't you import GlobalState instead of redefining it?

Reply:
Ack.

Reply:
@arajkumar I have updated PR with requested changes, Can you PTAL.